apps/sms/js/recipients.js

```diff
@@ -57,7 +64,10 @@
 
     if (this.isEmail) {
       this.className += ' email';
-    } else if (this.source === 'manual' && !rdigit.test(number)) {
+    } else if (
```

Comment:
Maybe we can join ```this.isQuestionable = opts.isQuestionable || false;``` and this code somehow that we assign this property only in one place?